tests/draft4/additionalItems.json

```diff
@@ -40,6 +40,11 @@
         },
         "tests": [
             {
+                "description": "fewer items is valid",
```

Comment:
The test below tests array with the same number of items. This tests for array with FEWER items. While it should pass and may seem redundant, some validators fail this test.